src/app/components/Device/DeviceList.tsx

```diff
@@ -76,7 +76,7 @@ export const DeviceTable = ({ devices, showFleet, refetch }: DeviceTableProps) =
                 <Td dataLabel="Creation timestamp">{device.metadata.creationTimestamp || '-'}</Td>
                 <Td dataLabel="Operating system">{device.status?.systemInfo?.operatingSystem || '-'}</Td>
                 <Td isActionCell>
-                  <ActionsColumn items={[deleteAction(device.metadata.name || '', deviceName)]} />
+                  <ActionsColumn items={[deleteAction({ resourceId: deviceName })]} />
```

Comment:
`deviceName` does not have to equal `device.metadata.name`
```suggestion
                  <ActionsColumn items={[deleteAction({ resourceId: device.metadata.name || '', resourceName: deviceName })]} />
```

Reply:
The variable `deviceName` is set to `device.metadata.name`, but I agree that we should pass both to the call to `deleteAction`.

So it would be something like:

```
            const deviceName = device.metadata.name as string;
            const displayName = device.metadata.labels?.displayName;
```
then

```
                  <ActionsColumn items={[deleteAction({ resourceId: deviceName, resourceName: displayName })]} />

```